README.md

```diff
@@ -1,17 +1,261 @@
+do our own thing. To do this, we need to hook up an event handler to the
+form's `submit` event, and then block the form from doing an HTML
+submit as it normally would.
+
+```erb
+# posts/new.html.erb
+<%= form_for(@post) do |f| %>
+  <div class="field">
+    <%= f.label :title %><br>
+    <%= f.text_field :title %>
+  </div>
+  <div class="field">
+    <%= f.label :description %><br>
+    <%= f.text_field :description %>
+  </div>
+  <div class="actions">
+    <%= f.submit %>
+  </div>
+<% end %>
+
+<script type="text/javascript" charset="utf-8">
+  $(function () {
+    $('form').submit(function(event) {
+      //prevent form from submitting the default way
+      event.preventDefault();
+      alert("we r hack3rz);
+    });
+  });
+</script>
+```
+
+We keep our regular `form_for` and just add an event listener for
+`$('form').submit()` to our document `ready()`.
+
+Next, we stop the form from actually submitting by calling
+`event.preventDefault();`. This is super important, otherwise all our
+hard work will be undone.
+
+Finally, we'll just toss up an `alert` to make sure we're on the right
+path.
+
+If we reload `/posts/new` now, and submit the form, we should get our
+`alert`, but no page refresh and no new post created. We've successfully
+hijacked the form `submit`! This is as close as we may ever come to
+being l33t.
+
+![crash override](http://i.giphy.com/13AN8X7jBIm15m.gif)
+
+Okay, that was fun, but let's do the real work. We need to get the form
+values and POST them to `/posts`, which is the route for creating a new
+post.
+
+```erb
+# posts/new.html.erb
+# ...
+<script type="text/javascript" charset="utf-8">
+  $(function () {
+    $('form').submit(function(event) {
+      //prevent form from submitting the default way
+      event.preventDefault();
+
+      var values = $(this).serialize();
+
+      var posting = $.post('/posts', values);
+
+      posting.done(function(data) {
+        // TODO: handle response
+      });
+    });
+  });
+</script>
+```
+
+In here, we're making use of the very handy [jQuery `serialize()`
+method](http://api.jquery.com/serialize/), which takes our form data and
+serializes it for us. Sure, we could individually select each
+form element and build our own JSON data, but we've done enough
+of that in this unit, so we've earned this shortcut!
+
+Next we use jQuery `post()`, much like we've been using `get()` to
+retrieve data. We pass it the URL and our `values`.
+
+Finally, we're using the `posting` object to specify what should happen
+when the AJAX request is `done`. This is where we'll need to handle the
+response.
+
+**Advanced:** The jQuery `post()` method returns a [jqXHR](http://api.jquery.com/jQuery.ajax/#jqXHR) object, which we're storing in our `posting` variable. These `jqXHR` objects implement the [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) interface, which is used for deferred or asynchronous operations.
```

Comment:
I really like this note here -- it does a good job of cluing the student in on what they might encounter in the future without overwhelming them.